### ISSUE TITLE
Request for RPM deliverables for F@H Arm Linux downloads

### ISSUE THREAD

**Issue by lamw** [FoldingAtHome/fah-issues#1613]
## Is your feature request related to a problem?

With the release of VMware's [ESXi on Arm](https://flings.vmware.com/esxi-arm-edition) distribution, I would like to build an Arm version of the F@H Virtual Appliance similar to that of the existing [F@H Virtual Appliance (x86)](https://flings.vmware.com/vmware-appliance-for-folding-home), which I was the creator of at VMware in collaboration with F@H organization.

----
## Describe the Feature

The Linux Arm distro that we would like to use is [VMware Photon OS](https://vmware.github.io/photon/), which is an RPM-based platform which we can not use with the existing F@H Arm deliverables which is in a .deb format. For x86, we simply use the F@H RPM packages. 

FYI - I did attempt to use `alien` to convert deb to rpm but didn't work and ideally, we have a native solution without having to go through such conversion

----
## Context

We have been getting a number of requests from VMware customers who have deployed ESXi on Arm and would like to run F@H as a workload. This would drive further usage of F@H for Arm and customers have been folding on behalf of Team VMware (ID: 52737)

----

**Comment by PantherX** [comment]
Hiya @lamw 

Thanks for your amazing contribution so far! I will ask around and see what I can find for you 😄

**Comment by gwildperson** [comment]
F@H does not support ARM clients.  Development work is being done by a variety of folks and I expect F@H will lend their support when a functional version is available.  Join the developer team on Discord.

**Comment by lamw** [comment]
@PantherX - I just noticed there's now beta builds for F@H Client supporting Arm, but looks like there's only .deb files. Any chance to produce an RPM like before?

**Comment by kbernhagen** [comment]
You might try using the ARM Linux tarball, built on Debian. 

https://download.foldingathome.org/releases/beta/fah-client/debian-stable-arm64/release/

**Comment by jcoffland** [comment]
I've published the .tar.bz2 files for Linux on the [beta release page](https://foldingathome.org/beta/).

**Comment by lamw** [comment]
Thanks @jcoffland - Do you know if the plan is to migrate/support the rest of the F@H components (fahviewer and fahcontrol) for Arm?

**Comment by lamw** [comment]
Also, while the .tar.bz2 works on non-debian systems, I found that it doesn't include what the standard installer includes which are the .service files and directory structures, just the client only. It would be ideal that we have same install experience whether its via package manager or manual install

**Comment by jcoffland** [comment]
Good point. I will add those things to the tar.bz2.  fahviewer and fahcontrol are not needed in v8.  See the User's Guide.
